scripts/shared/clusters.sh

```diff
@@ -91,6 +95,10 @@ function create_kind_cluster() {
         kind delete cluster --name=${cluster}
         return 1
     fi
+
+    if [[ $olm = "true" ]]; then
+      deploy_olm
+    fi
```

Comment:
```suggestion
    [[ $olm != "true" ]] || deploy_olm
```

I prefer this syntax (alternatively, you can stick it in deploy_olm right at the start).

Should the OLM deployment fail though, would you want to retry the cluster (as in deploy_cni above)?

Reply:
Humm, good question. I haven't seen it fail yet, and it seems like it should be more reliable than Weave, so I'm not sure retries are super important. Idk though, I'm open to recommendations.

Reply:
Well just because it hasn't happened yet doesn't mean it can't :)

Perhaps have a function 'deploy_cluster_capabilities'  or something, which will be called in line 88 (instead of deploy_cni) and in turn it'll call `deploy_cni` and `deploy_olm`.

This way if that function fails, it would destroy the cluster and retry..

Reply:
@mkolesnik okay, good ideas, done in the latest. :)